### ISSUE TITLE
Unable to install HARE using RPM

### ISSUE THREAD

**Issue by TechWriter-Mayur** [Seagate/cortx-hare#1607]
Hi Team,

I am following the HARE User Guide and tried to install/configure the HARE using RPM but I am facing the below issue:

```
[root@localhost hare]# sudo yum -y install cortx-motr cortx-motr-devel
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: centos-hcm.viettelidc.com.vn
 * extras: centos.excellmedia.net
 * updates: centos-hcm.viettelidc.com.vn
No package cortx-motr available.
No package cortx-motr-devel available.
Error: Nothing to do
```
I think the RPM package is not available in the location but there is no instruction to create RPM package? Is it removed from the repo? or do we need to create it?

**Comment by vaibhavparatwar** [comment]
This issue is same as #1589  . But as the commands suggeset, this is issue with installing motr using RPMs and not HARE issue.

**Comment by knekrasov** [comment]
Resolved by #1610
